pkg/omf/functions/packages/omf.packages.list.fish

```diff
@@ -72,6 +88,6 @@ function omf.packages.list -a option type
   case "-i" "--installed"
     __omf.packages.list.installed $type
   case "*"
-    __omf.packages.list $type
+    __omf.packages.list.installed $type
```

Comment:
if this is the case, couldn't we just get rid of the case before (or add `"*"` to the matching case and combine the two)?